location/src/androidLibMain/kotlin/location/AndroidLocationFlowable.kt

```diff
@@ -18,13 +18,14 @@ Copyright 2019 Splendo Consulting B.V. The Netherlands
 */
 
 import com.google.android.gms.location.*
+import com.splendo.kaluga.base.ContextProvider
 import com.splendo.kaluga.log.debug
 import kotlinx.coroutines.*
 import kotlinx.coroutines.tasks.await
 
 data class FusedLocationProviderHandler(
     private val locationFlowable: LocationFlowable,
-    private val fusedLocationProviderClient: FusedLocationProviderClient
+    private val fusedLocationProviderClient: FusedLocationProviderClient = LocationServices.getFusedLocationProviderClient(ContextProvider.context)
```

Comment:
@thoutbeckers would it be ok to add this since now we have access to context?

Reply:
This class is used internally by the class below (probably needs `internal` on it) so I think this will not do anything.

Reply:
👍